### PR TITLE
[FIX] mass_mailing_custom_unsubscribe: Pivot error

### PR DESCRIPTION
**Pull request by victoralmau**
Solved error in pivot view.

@Tecnativa TT26101

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: mail_debrand/i18n/it.po [hunk 1]
```diff
@@ -6,38 +6,40 @@ msgid ""
 msgstr ""
```
Comment (pedrobaeza) [inline mail_debrand/i18n/it.po]: Why is this change here?

Comment (victoralmau) [reply mail_debrand/i18n/it.po]: Very strange, obviously .po file not necessary to modify. I will check my commit again and solve it.

**Comment by pedrobaeza** [comment]
Travis is red

**Comment by victoralmau** [comment]
> Travis is red

Locked by: https://github.com/OCA/social/pull/630
